src/App.js

```diff
@@ -9,7 +12,10 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <>
+    <h2>React Article</h2>
+    <Article title={article.title} date={article.date} text={article.text} />
```

Comment:
https://kevinyckim33.medium.com/jsx-spread-operator-component-props-meaning-3c9bcadd2493

```suggestion
    <Article {...article} />
```